### PR TITLE
GenericSQL - fix ssl connection

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23743

## Description
Fixed the option to connect with ssl 
in order to make it user friendly the option was added as a button

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py [hunk 1]
```diff
@@ -63,6 +64,11 @@ def _create_engine_and_connect(self) -> sqlalchemy.engine.base.Connection:
             elif self.connect_parameters and self.dialect != "Microsoft SQL Server":
                 # a "?" was already added when the driver was defined
                 db_preferences += f'?{self.connect_parameters}'
+
+            if self.ssl_connect:
```
Comment (bakatzir) [inline Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:68]: can you have connect_parameteres and ssl_connect?
If so, this code will not allow it.

Comment (ohaim1008) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:68]: yes, tried and it looks ok, I tried to add the ssl dict to the connect_parameters in the way we currently use - "?arg1=something" but it doesn't support nested dictionaries. 
why won't it? the other connection_args are in the db_preferences string.

Comment (bakatzir) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:68]: You are right. can we have 1 return statement, and `ssl_connection` would be an empty dict if the `ssl_connect` is not true?
It feels redundant to have different return statement for this flow.

Comment (ohaim1008) [reply Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py:69]: sure, it will just be empty.
change was made.

**Comment by bakatzir** [comment]
@kirbles19 we have added a new parameter to the integration, and would love your help.

**Comment by kirbles19** [comment]
@ohaim1008 Done

**Pull request merged**